account/src/use-case/createUserToken.js

```diff
@@ -0,0 +1,20 @@
+import { comparePassword } from "../helpers/passwords.js";
+import { generateToken } from "../helpers/token.js";
+import {findUserByEmail } from "../repositories/accountRepository.js";
+
+export async function createUserTokenUseCase(email, password){
+    const userFound = await findUserByEmail(email);
+
+    if(!userFound){
+        return null;
+    }
+
+    const passwordIsMatch = await comparePassword(password, userFound.password);
+
+    if (passwordIsMatch) {
+        return generateToken(userFound._id);
+
+    }
```

Comment:
i suggest you always put the happy flow at the end

Reply:
adjusted.